### DISCUSSION TITLE
Support of additional marker types for Graphs

### DISCUSSION THREAD

**Discussion by andreashofm** [grafana/grafana#38166] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Currently, the plot type "Graph" does only support (filled) point markers. It would be great if additional markers like squares, triangles, stars, plus, etc. could be added.

**Why is this needed**:
Having different marker types would be beneficial when superimposing several data streams as well as for highlighting purposes.

**Comment by hugohaggmark** [comment]
@dprokop is this something we're thinking about supporting in the new Graph?

**Comment by dprokop** [comment]
@andreashofm thank you for this feature request. I have added it to the new graph panel backlog and we will try to include this feature in the new graph panel.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by v-avdeev** [comment]
In my case it would be nice to have this feature to draw trading signals on the candlestick panel:
![image](https://user-images.githubusercontent.com/3287643/232238146-8814c98f-ade6-40b7-a303-ec192c3232d8.png)

**Comment by kevinyin9** [comment]
Hi. Is there any update about this feature? Look forward to it!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Comment by mmonaco** [comment]
Can you reopen the original issue https://github.com/grafana/grafana/issues/28528 that this discussion was based on?

**Reply by gelicia** [reply]
I was unable to open that issue, but I created a new issue from the discussion - https://github.com/grafana/grafana/issues/88634
